app/controllers/twilio_controller.rb

```diff
@@ -1,14 +1,41 @@
 class TwilioController < ApplicationController
   skip_before_action :verify_authenticity_token
 
+  def receive_text
+    recieve_phone(true, params)
```

Comment:
There is a typo for the `recieve_phone` method. It should be `receive_phone`. A quick auto-refactor from RubyMine should fix this :)